packages/forms/src/UIForm/fields/Comparator/Comparator.component.js

```diff
@@ -92,27 +123,35 @@ class Comparator extends React.Component {
 		const symbols = (schema.options && schema.options.symbols) || {};
 		return this.getOperatorSchema().titleMap.map(({ value }) => {
 			const titles = map.find(m => m.value === value);
-			const title = titles ? `${titles.symbol} (${titles.name})` : value;
+			let title;
+			if (titles) {
+				title = titles.symbol ? `${titles.symbol} (${titles.name})` : titles.name;
+			} else {
+				title = value;
+			}
```

Comment:
Should it be nicer with a little function to do that ?

Reply:
Sure, I will fix it